### PR TITLE
Add page documenting banlist functionality

### LINKED ISSUE #99: Document new banlist functionality
- https://github.com/R2Northstar/NorthstarLauncher/pull/227
- https://github.com/R2Northstar/NorthstarLauncher/pull/228
- https://github.com/R2Northstar/NorthstarLauncher/pull/233

### LINKED ISSUE #141: Document banlist comments
https://github.com/R2Northstar/NorthstarLauncher/pull/227

### PR DESCRIPTION
**Pull request by itscynxx**
Adds docs under `server settings` of how `banlist.txt` works

Closes #99
Closes #141

### CODE REVIEW

File: docs/hosting-a-server-with-northstar/server-settings/banlist.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+## Unbanning
+
+When unbanning a player in game using the `unban` command in console, their UID will be commented out in the file, as well as a comment next to their UID displaying the unban date.
+
+The `unban` command is used alongisde the UID of the player you want to unban (e.g. `unban 1111111111`).
+
+## Banlist reloading
+
+The banlist automatically reloads when a player joins, so there's no need to worry about if the list will be up to date, or if you need to manually reload it.
```
Comment (GeckoEidechse) [inline docs/hosting-a-server-with-northstar/server-settings/banlist.md]: This section might not even be necessary ^^

File: docs/hosting-a-server-with-northstar/server-settings/banlist.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+4444444444
+```
+
+This file also supports the use of comments with the `#` symbol:
+
+```
+# This is a comment
+1111111111
+
+4444444444
```
Comment (GeckoEidechse) [inline docs/hosting-a-server-with-northstar/server-settings/banlist.md]: I think we can already have an example of inline comment here ^^

```suggestion
4444444444 # Banned for being uncool
```

File: docs/hosting-a-server-with-northstar/server-settings/banlist.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+```
+# This is a comment
+1111111111
+    2222222222
+
+ 3333333333 # Wow a second comment!
+4444444444
+```
+
```
Comment (GeckoEidechse) [inline docs/hosting-a-server-with-northstar/server-settings/banlist.md]: I think we can remove this part tbh. I don't think it's important to mention how strict or lenient the syntax is.
Also helps with keeping the section concise and straight to the point :D

```suggestion

```

**Review by GeckoEidechse — COMMENTED** [review]
Damn that was a fast PR, haha. Thanks for making it. Just a few small things and then it should be good to merge :D

**Comment by itscynxx** [comment]
GitHub was being annoying with allowing me to commit via suggestion, feel free to add yourself as a co author @GeckoEidechse

**Review by GeckoEidechse — APPROVED** [review]
Changes from review have been addressed. Should be good to merge :D

**Pull request merged**